README.md

```diff
@@ -17,7 +17,7 @@
 
 ## Application Page url
 
```

Comment:
- [ ] Users should be able to edit task descriptions. I couldn't find any function related to this.

Reply:
Addressed this